backend/utils.py

```diff
@@ -86,17 +86,10 @@ def format_non_streaming_response(chatCompletion, history_metadata, message_uuid
     if len(chatCompletion.choices) > 0:
         message = chatCompletion.choices[0].message
```

Comment:
Can we support both formats for the output for the moment? In case any user pulls in the code change, but has the env variable set back to the older API version

Reply:
Actually I take it back - since /extensions/chat/completions is removed, we can't support the old API version at all. Let's add a fail-fast check at the beginning of the request handler for the old API version